setup.py

```diff
@@ -44,7 +44,7 @@ def read_version():
     "werkzeug==0.15.5",
     "paramiko==2.4.2",
     "psutil==5.4.8",
-    "protobuf>=3.1",
+    "protobuf==3.11",
```

Comment:
I think this version pin change should be done in python sdk. since the error is coming from there: https://github.com/aws/sagemaker-python-sdk/blob/master/setup.py#L39

Reply:
https://github.com/aws/sagemaker-containers/blob/master/setup.py#L94

This might've overwrite the pinned 3.11 version?